contents.md

```diff
@@ -33,16 +33,44 @@ at home to participate offline stage. Learn more in [README](README.md).
 
 ## Stage 1. The Show Must Go On
 
-__Under revision__
+### Main thread
 
+ 1. Intro to meet-up backed course stage
+ 1. Self-study stage practical tasks retrospection
  1. Mastering HTML 5 forms
- 1. Responsive Layouts
- 1. Backend API (Weather App)
- 1. Pizza App: Login and Registration
- 1. Pizza App: Pizza Creation
- 1. ReactJS Weather App
- 1. React + Redux
+ 1. Modern Web App and Web  Frameworks
+ 1. Component-based App Architecture (Weather App)
+ 1. Routing (Collection Browsing App)
+ 1. Authentication and Authorization (Chat App)
+ 1. Websockets (Chat App)
+ 1. Canvas API (Puzzle Game)
+ 1. Framework/Library (_TBD_)
+ 
+### Supplementary workshops
 
+ 1. JS Asynchronicity
+ 1. JS tricks, best practices
+ 1. JS prototypes
+ 1. Tooling
+ 1. Advanced git
+ 1. Whatever students want to try (within FrontEnd Universe)
+ 
+### Supplementary lectures
+
+ 1. Design Patterns Classics
```

Comment:
Maybe we should add some short description and keep only one version of all themes and topics? 

From https://github.com/kottans/frontend/pull/176

Reply:
Also,  we will share this description with students, so it will be good to provide a description of what "Supplementary" is.

Reply:
My feeling is that any details are more for mentors rather for beginners and might cause stack overflow at this stage.
Can we add these listings now to give the students feeling that the course contents is more or less aligned?
We could decide if this document is a proper place for details and if we are positive than supplement this doc with more details via new PR.
What do you think?

Reply:
I think that we should add the link to issue with a list, because some students can ask questions about the meaning of this topics.